xcc/commands.py

```diff
@@ -257,18 +257,19 @@ def get_job(
 
 
 @beautify
-def list_jobs(limit: int = 5, ids: List[str] = None) -> Sequence[Mapping]:
+def list_jobs(limit: int = 5, ids: List[str] = None, status: str = None) -> Sequence[Mapping]:
     """Lists jobs submitted to the Xanadu Cloud.
 
     Args:
         limit (int): Maximum number of jobs to display.
         ids (List[str], optional): IDs of the jobs to display. If at least one
             ID is specified, the limit flag will be set to the number of IDs.
+        status (str): Status parameter used to filter returned jobs based on given status.
```

Comment:
I think the wording here can be simplified. Also, we should document that the flag is optional:
```suggestion
        status (str, optional): If specified, filter jobs by the given status.
```